tests/conftest.py

```diff
@@ -4,15 +4,33 @@
 from ragger.firmware import Firmware
 from ragger.backend import SpeculosBackend, LedgerCommBackend, LedgerWalletBackend
 from ragger.navigator import NanoNavigator, StaxNavigator
-from ragger.utils import app_path_from_app_name
 
+# To move into ragger with the unified conftest
+def is_root(path_to_check: Path) -> bool:
+    return (path_to_check).resolve() == Path("/").resolve()
 
-# This variable is needed for Speculos only (physical tests need the application to be already installed)
-# Adapt this path to your 'tests/elfs' directory
-APPS_DIRECTORY = (Path(__file__).parent / "elfs").resolve()
```

Comment:
can you also remove the build_all Makefile target that was create for this and is not necessary anymore?

Reply:
Done, I didn't even know this existed